examples/models/llama/export_llama_lib.py

```diff
@@ -684,6 +689,15 @@ def _validate_args(args):
     if args.num_sharding > 0 and not args.qnn:
         raise ValueError("Model shard is only supported with qnn backend now.")
 
+    if args.use_shared_embedding:
+        if not (
+            args.embedding_quantize is not None
+            and args.embedding_quantize.startswith("torchao:")
```

Comment:
```suggestion
    if args.use_shared_embedding:
        and (
            args.embedding_quantize is None
            or not args.embedding_quantize.startswith("torchao:")
```

nit: nested conditionals into an error